docs/observations.rst

```diff
@@ -13,3 +13,9 @@ GRDC
 ----
 
 The `Global Runoff Data Centre <https://grdc.bafg.de/>`_ provides discharge data for a large number of stations around the world. In eWaterCycle we support GRDC data. This is not downloaded automatically, but required to be present on the infrastructure where the eWaterCycle platform is deployed. By special permission from GRDC our own instance contains data from the ArcticHYCOS and GCOS/GTN-H, GTN-R projects.
+
+Caravan
+-------
+
+The `Caravan <https://doi.org/10.1038/s41597-023-01975-w>`_ dataset contains river discharge for each of its basins.
+Observations can be retrieved using the :py:func:`ewatercycle.observation.caravan.get_caravan_data` function.
```

Comment:
Could be nice to link to https://www.ewatercycle.org/caravan-map/ so it's easy to find basins.